doc/plugins/zipkin-cn.md

```diff
@@ -49,7 +51,9 @@ curl http://127.0.0.1:9080/apisix/admin/routes/1  -H 'X-API-KEY: edd1c9f034335f1
     "plugins": {
         "zipkin": {
             "endpoint": "http://127.0.0.1:9411/api/v2/spans",
-            "sample_ratio": 1
+            "sample_ratio": 1,
+            "service_name": "APISIX-IN-SG",
+            "server_addr": "8.8.8.8"
```

Comment:
Is it the server's address that needs to be filled in here, or is it the DNS address?

Reply:
I just want to make an example to user  that they can specific any IP address here

Reply:
how about change it into `1.2.3.4`